source/direct_debit/index.html.md.erb

```diff
@@ -255,96 +133,197 @@ Required arguments:
  "amount"
  "description"
  "reference"
- "agreement_id"
- }
- ```
+ "mandate_id"
+}
+```
 
 Once a mandate has been set up and a mandate ID is available, you can make an
 API call to collect a payment against the mandate.
 
-You do not need to wait until the mandate is confirmed to send a collection
-request to GOV.UK Pay.
+You do not need to wait until the mandate is confirmed to send a collection request to GOV.UK Pay.
 
-When collecting a payment against a mandate, you must [supply a
-Reference number](#reference-numbers) and description.
-
-If a mandate does not exist prior to a payment request, it takes 4 working
+If a mandate does not exist prior to a payment request, it takes 3 working
 days to collect a payment.
 
-If a mandate has already been set up, it takes 3 working days to collect a
-payment each time a subsequent one is requested. It then takes a further 2
-working days before the payment reaches the service’s bank account.
+If a mandate has already been set up, it takes 2 working days to collect a
+payment each time a subsequent one is requested. It then takes a further 1
+working day before the payment reaches the service’s bank account.
 
-GOV.UK Pay will send an email to the user giving them 3 working days' advance
+GOV.UK Pay will send an email to the user giving them 2 working days' advance
 notification that payment will be collected against their mandate. The email will contain the mandate reference number.
 
 You can collect at any time interval. You may collect between £1 and £5000 each
```

Comment:
I think “at any time interval” could just be “whenever you want” (or something).

Reply:
Tweaked  in https://github.com/alphagov/pay-tech-docs/pull/298/commits/562c05d6cc099a67190a4bfe298a6acc6291794c - thanks!